schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+-- create patients table
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
```

Comment:
- [ ] Kindly ensure you use the same data type shown in the ERD. `Name` should be `varchar` and not `text`. Deviating from the ERD can introduce inconsistencies and make it harder to understand and maintain the database structure.